vllm/config/__init__.py

```diff
@@ -851,7 +852,8 @@ def maybe_pull_model_tokenizer_for_runai(self, model: str,
 
         # Only download tokenizer if needed and not already handled
         if is_runai_obj_uri(tokenizer):
-            object_storage_tokenizer = ObjectStorageModel()
+            directory = hashlib.sha256(str(tokenizer).encode()).hexdigest()[:8]
```

Comment:
create a util function for these hashing specifics that is shared between tokenizer and model?